### PR TITLE
Create Etown_Wittmeyer.yaml and ITT_Zhong.yaml

### PR DESCRIPTION
**Pull request by rachellombardi**

### CODE REVIEW

**Comment by mwestphall** [comment]
Hi @jstathas , this topology change creates a new organization "Elizabethtown College". Do we have the project office's approval to add Elizabethtown College as an organization?

**Comment by brianhlin** [comment]
Pinging @jstathas' inbox againregarding new institution `Elizabethtown College`

**Comment by brianhlin** [comment]
@rachellombardi can you confirm the institution's location?

**Comment by rachellombardi** [comment]
Confirmed.

File: projects/Etown_Wittmeyer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Description: I am interested in examining experience-dependent neuroplasticity and individual differences in humans as it pertains to learning and memory.  In particular, I analyze structural magnetic resonance imaging (sMRI) data from various neuroimaging data-sharing platforms to explore changes in gray matter across learning and/or correlate learning performance with various cognitive and demographic factors.
+Department: Psychology
+FieldOfScience: Psychology and Life Sciences
+Organization: Elizabethtown College
+PIName: Jennifer Legault Wittmeyer
+
+ID: '591'
```
Comment (brianhlin) [inline projects/Etown_Wittmeyer.yaml:7]:
```suggestion
```

ID no longer required!

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**